Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM node:5.11-wheezy
```

Comment:
By removing 5.11-wheezy, I think we are guaranteed to get the latest, should we change it to that? is it good practice?